apps/cli-daemon/src/app/workspace/workspace.controller.ts

```diff
@@ -102,4 +115,24 @@ export class WorkspaceController {
 
     return target;
   }
+
+  private ngNewArgsFromDto(dto: WorkspaceCreateDto): string[] {
+    const { name, options } = dto;
+    const args: string[] = [name];
+
+    const ngNewOptions = options.map((option) => {
+      // If the option value is undefined, assume it's a boolean flag and include only the option name
+      const optionValue = option.value !== undefined ? ` ${option.value}` : '';
+
+      // If the option is interactive, set it to false
+      if (option.name.includes('--interactive')) {
+        return `--interactive=false`;
+      }
+
+      // Otherwise, include the option name and value (if any)
+      return `${option.name}${optionValue}`;
```

Comment:
no space or = between?

Reply:
There's a space in the value itself, because if there's no value there's no need for the space

Reply:
oh I see it now, ok